es6-features-practise/test/iterators.js

```diff
@@ -122,23 +129,23 @@ describe('A simple iterable without items inside, implementing the right protoco
   });
   describe('using the iterable', function() {
     it('it contains no values', function() {
-      let values;
+      let values = 0;
```

Comment:
`let values = ''`

in your case, `values` will be `0`

Reply:
Why is this better?